Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -12480,7 +12528,7 @@ def build_pbf_xpath(name, pre_post, element_to_change=None, filters: dict = None
 
     if query:
         _xpath = f"{_xpath}/rules/entry[{query}]"
-    elif xpath_filter := build_xpath_filter(name, filters):
+    elif xpath_filter := build_xpath_filter(name_match=name, filters= filters):
```

Comment:
```suggestion
    elif xpath_filter := build_xpath_filter(name_match=name, filters=filters):
```
should be without space, my bad from last review.